src/translate.rs

```diff
@@ -1162,38 +1176,67 @@ impl FromGlib<i32> for Option<u64> {
     }
 }
 
-/// Translate from a pointer type without taking ownership, transfer: none.
+/// Translate from a pointer type which is annotated with `transfer: none`.
+/// The resulting value is referenced at least once, by the bindings.
```

Comment:
Maybe we need separate description from 2 sides: from users (this phrase), and for implementers (next phrase) for all aspects.
Analog for this phrase for impls: "For referenced types implementation need to use `ref_sink` instead `ref` to correctly handle floating reference,
for copy types implementation need to make `copy`"

Reply:
Or not mention "implementers" side at all

Reply:
"`ref_sink` (if it present for type) instead .." or so